### ISSUE TITLE
Accuracy of the released model does not match what you report

### ISSUE THREAD

**Issue by yxchng** [wy1iu/sphereface#93]
```
fold	ACC
----------------
1	99.33%
2	99.33%
3	99.00%
4	99.50%
5	98.67%
6	99.33%
7	99.17%
8	99.00%
9	100.00%
10	99.33%
----------------
AVE	99.27%
```

This is the result I get by running your code and model. You put the released model as model 3 which you mentioned in the repo to have the result of 99.3%. Is it a mistake?

![screenshot from 2018-05-29 14-46-59](https://user-images.githubusercontent.com/10518587/40642263-24a1aaa2-634f-11e8-8977-ad696c4b4f61.png)

Also what you mean by "going through the pipeline 5 times"? Is it that you train 5 times and get 5 models?

**Comment by wy1iu** [comment]
We obtained 99.3% accuracy for the pretrained model in our experiment. Our reported result is not a mistake.

The reason that you only get 99.27% may vary. Usually, it is due to some mismatches in the pre-processing steps (detection and alignment), not in the recognition part.

**Comment by yxchng** [comment]
@wy1iu I ran your preprocessing code: face_detect_demo.m and face_align_demo.m. How can the result be different when I ran your code?

**Comment by wy1iu** [comment]
I am not quite sure what causes the 0.03% accuracy drop in your case. The reason can vary. But you can retrain the model from scratch using our code. It is not difficult to reach 99.3%.

**Comment by zeakey** [comment]
@yxchng Bingo!
```
fold	ACC
----------------
1	99.33%
2	99.33%
3	99.00%
4	99.50%
5	98.67%
6	99.33%
7	99.17%
8	99.00%
9	100.00%
10	99.33%
----------------
AVE	99.27%
```
I reproduce the **99.27%**.
Landmarks and cropped images are obtained with the author provided code.

What a joke.

**Comment by zeakey** [comment]
Since all the computation are performed on the GPUs, my system information that may affects the performance is:
* cuda version:  `ver. 9.1`
* cudnn version: `ver. 7.0.5`
The images are read by Matlab, the version of my matlab is `R2015b`.

**Comment by wy1iu** [comment]
@zeakey We used cuda8 w/o cudnn. I think the problem lies in the preprocessing part, since you use the pretrained model and still can not get 99.27%. You should pay attention to the preprocessing steps.

**Comment by nile649** [comment]
What is the accuracy you should get in epoch 1, I am using pytorch so everything is different. Any help would be awesome
